go/types/xp_test.go

```diff
@@ -72,6 +80,6 @@ func (suite *testSuite) TestTypes() {
 }
 
 // Called from "go test"
-func TestSuite(t *testing.T) {
+func TestPlatformSuite(t *testing.T) {
```

Comment:
TestSuite is too generic... The name of this is needs to be `Test.+` but since the types package is so large calling it TestSuite seems not specific enough.